### ISSUE TITLE
NoMethodError in Layout/AlignHash cop on version 0.52.0

### ISSUE THREAD

**Issue by nilsding** [bbatsov/rubocop#5239]
After upgrading Rubocop from version `0.51.0` to `0.52.0`, the `Layout/AlignHash` cop started to fail with a `NoMethodError` on a specific part in my code:

``` ruby
some_method(
  **Hashie.symbolize_keys(a_hash).merge(
    key: :value
  )
)
```

The problem here is the double splat.  If I remove this or change it to a single splat, the cop runs just fine.

--------

## Expected behavior

Behave like `0.51.0` and do not error on the double splat:

```
% rubocop -V
0.51.0 (using Parser 2.4.0.2, running on ruby 2.4.2 x86_64-darwin16)
% rubocop snippet.rb
For /Users/nilsding: configuration from /Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.51.0/config/default.yml
Inheriting configuration from /Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.51.0/config/enabled.yml
Inheriting configuration from /Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.51.0/config/disabled.yml
Inspecting 1 file
Scanning /Users/nilsding/snippet.rb
.

1 file inspected, no offenses detected
Finished in 0.06374499999219552 seconds
```

## Actual behavior

It errors on `0.52.0`:

```
% rubocop -d snippet.rb
For /Users/nilsding: configuration from /Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/config/default.yml
Inheriting configuration from /Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/config/enabled.yml
Inheriting configuration from /Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/config/disabled.yml
Inspecting 1 file
Scanning /Users/nilsding/snippet.rb
An error occurred while Layout/AlignHash cop was inspecting /Users/nilsding/snippet.rb:2:2.
undefined method `hash_rocket?' for nil:NilClass
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/cop/layout/align_hash.rb:173:in `alignment_for'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/cop/layout/align_hash.rb:152:in `check_pairs'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/cop/layout/align_hash.rb:124:in `on_hash'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/cop/commissioner.rb:44:in `block (2 levels) in on_hash'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/cop/commissioner.rb:109:in `with_cop_error_handling'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/cop/commissioner.rb:43:in `block in on_hash'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/cop/commissioner.rb:42:in `each'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/cop/commissioner.rb:42:in `on_hash'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/ast/traversal.rb:101:in `block in on_send'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/ast/traversal.rb:99:in `each'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/ast/traversal.rb:99:in `each_with_index'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/ast/traversal.rb:99:in `on_send'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/cop/commissioner.rb:48:in `on_send'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/ast/traversal.rb:12:in `walk'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/cop/commissioner.rb:60:in `investigate'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/cop/team.rb:114:in `investigate'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/cop/team.rb:102:in `offenses'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/cop/team.rb:44:in `inspect_file'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/runner.rb:258:in `inspect_file'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/runner.rb:205:in `block in do_inspection_loop'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/runner.rb:237:in `block in iterate_until_no_changes'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/runner.rb:230:in `loop'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/runner.rb:230:in `iterate_until_no_changes'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/runner.rb:201:in `do_inspection_loop'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/runner.rb:111:in `block in file_offenses'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/runner.rb:121:in `file_offense_cache'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/runner.rb:109:in `file_offenses'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/runner.rb:100:in `process_file'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/runner.rb:78:in `block in each_inspected_file'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/runner.rb:75:in `each'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/runner.rb:75:in `reduce'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/runner.rb:75:in `each_inspected_file'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/runner.rb:67:in `inspect_files'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/runner.rb:39:in `run'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/cli.rb:128:in `execute_runner'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/cli.rb:60:in `execute_runners'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/lib/rubocop/cli.rb:31:in `run'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/bin/rubocop:13:in `block in <top (required)>'
/Users/nilsding/.rvm/rubies/ruby-2.4.2/lib/ruby/2.4.0/benchmark.rb:308:in `realtime'
/Users/nilsding/.rvm/gems/ruby-2.4.2/gems/rubocop-0.52.0/bin/rubocop:12:in `<top (required)>'
/Users/nilsding/.rvm/gems/ruby-2.4.2/bin/rubocop:23:in `load'
/Users/nilsding/.rvm/gems/ruby-2.4.2/bin/rubocop:23:in `<main>'
/Users/nilsding/.rvm/gems/ruby-2.4.2/bin/ruby_executable_hooks:15:in `eval'
/Users/nilsding/.rvm/gems/ruby-2.4.2/bin/ruby_executable_hooks:15:in `<main>'
.

1 file inspected, no offenses detected

1 error occurred:
An error occurred while Layout/AlignHash cop was inspecting /Users/nilsding/snippet.rb:2:2.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
Mention the following information in the issue report:
0.52.0 (using Parser 2.4.0.2, running on ruby 2.4.2 x86_64-darwin16)
Finished in 0.13664099999004975 seconds
```

## Steps to reproduce the problem

1. Create a Ruby file with these contents:
``` ruby
some_method(
  **Hashie.symbolize_keys(a_hash).merge(
    key: :value
  )
)
```
2. Run `rubocop` on this file with the default settings.

## RuboCop version

```
$ rubocop -V
0.52.0 (using Parser 2.4.0.2, running on ruby 2.4.2 x86_64-darwin16)
```

**Comment by nilsding** [comment]
That was quick.  Thanks! 🎉
